apps/nextjs-website/react-components/types/Stats/Stats.types.ts

```diff
@@ -0,0 +1,13 @@
+export interface StatsItemProps {
+  readonly title: string;
+  readonly description?: string;
+  readonly iconURL?: string;
+}
+
+export interface StatsProps {
+  readonly eyelet?: string;
+  readonly title: string;
+  readonly body?: string;
+  stats: ReadonlyArray<StatsItemProps>;
```

Comment:
```suggestion
  items: ReadonlyArray<StatsItemProps>;
```
Rename this as well, following previous suggestions.